app/models/project.rb

```diff
@@ -645,7 +645,7 @@ def check_status
     response = Typhoeus.get(url)
     if platform.downcase == "packagist" && [302, 404].include?(response.response_code)
       update_attribute(:status, "Removed")
-    elsif platform.downcase == "go" && [400, 404].include?(response.response_code)
+    elsif platform.downcase == "go" && [302, 400, 404].include?(response.response_code)
```

Comment:
Based off the logic in the description, should all packages that redirect be marked as "Removed"?

Also, maybe not for this PR, but thoughts about moving the platform specific statusing logic into their own classes vs a giant if/elsif/else statement?